src/Power_Up_Sequence.md

```diff
@@ -312,19 +312,19 @@ Name          | Address | DMG0     | DMG / MGB | SGB / SGB2 | CGB / AGB
 [`OBP1`]      | $FF49   | ??[^obp] | ??[^obp]  | ??[^obp]   | ??[^obp]
 [`WY`]        | $FF4A   | $00      | $00       | $00        | $00
 [`WX`]        | $FF4B   | $00      | $00       | $00        | $00
-[`KEY1`]      | $FF4D   | $FF      | $FF       | $FF        | $FF
-[`VBK`]       | $FF4F   | $FF      | $FF       | $FF        | $FF
-[`HDMA1`]     | $FF51   | $FF      | $FF       | $FF        | $FF
-[`HDMA2`]     | $FF52   | $FF      | $FF       | $FF        | $FF
-[`HDMA3`]     | $FF53   | $FF      | $FF       | $FF        | $FF
-[`HDMA4`]     | $FF54   | $FF      | $FF       | $FF        | $FF
-[`HDMA5`]     | $FF55   | $FF      | $FF       | $FF        | $FF
-[`RP`]        | $FF56   | $FF      | $FF       | $FF        | $FF
-[`BCPS`]      | $FF68   | $FF      | $FF       | $FF        | ??[^compat]
-[`BCPD`]      | $FF69   | $FF      | $FF       | $FF        | ??[^compat]
-[`OCPS`]      | $FF6A   | $FF      | $FF       | $FF        | ??[^compat]
-[`OCPD`]      | $FF6B   | $FF      | $FF       | $FF        | ??[^compat]
-[`SVBK`]      | $FF70   | $FF      | $FF       | $FF        | $FF
+[`KEY1`]      | $FF4D   | ---      | ---       | ---        | N/A or $7E[^compat]
```

Comment:
I think it'd be more appropriate to make an extra footnote:
```md
[^cgb_only]: These registers are only available in CGB Mode, and will read \$FF in Non-CGB Mode.
```
...and this would thus become:
```suggestion
[`KEY1`]      | $FF4D   | ---      | ---       | ---        | $7E[^cgb_mode]
```

I think that wording is clearer.

Reply:
Should I apply these to the HDMA regs as well? I will just in case to save time, and revert them if not good.

Reply:
Yeah, I don't see why not.